quantum/process_keycode/process_unicode_common.c

```diff
@@ -139,26 +156,44 @@ __attribute__((weak)) void unicode_input_cancel(void) {
             unregister_code(UNICODE_KEY_MAC);
             break;
         case UC_LNX:
-            tap_code(KC_ESC);
+            tap_code(KC_ESCAPE);
             if (unicode_saved_caps_lock) {
-                tap_code(KC_CAPS);
+                tap_code(KC_CAPSLOCK);
             }
             break;
         case UC_WINC:
-            tap_code(KC_ESC);
+            tap_code(KC_ESCAPE);
             break;
         case UC_WIN:
             unregister_code(KC_LALT);
+#ifdef UNICODE_UC_WIN_USE_NUMPAD_IF_POSSIBLE
+            if (!unicode_saved_num_lock) {
+                tap_code(KC_NUMLOCK);
+            }
+#endif
             break;
     }
 
     set_mods(unicode_saved_mods);  // Reregister previously set mods
 }
 
```

Comment:
The linter's going to complain without this.

```suggestion

// clang-format off

```